l10n/en/mozorg/home-new.ftl

```diff
@@ -60,7 +59,7 @@ home-so-what-is-mozilla = So, what is { -brand-name-mozilla }?
 
 # Variables
 #   $ventures - link to https://mozilla.vc/
-#   $mozai - link to https://www.mozilla.ai/
+#   $mozai - link to https://mozilla.ai/
 home-at-its-core = At its core, { -brand-name-mozilla } is an activist organization led by the { -brand-name-mozilla-foundation } that makes change in the world through a variety of ventures including { -brand-name-mozilla-corporation }, MZLA, <a {$ventures}>{ -brand-name-mozilla-ventures }</a> and <a {$mozai}>{ -brand-name-mozilla-ai }</a>. How are we different? Because we’re mission-driven, it means we have the freedom to make all of our decisions based on what’s best for the internet and for everyone online, not based on the demands of shareholders — we don’t actually have any of those.
```

Comment:
There's one OQ from https://github.com/mozilla-l10n/www-l10n/pull/404#discussion_r1614788274: 

```suggestion
home-at-its-core = At its core, { -brand-name-mozilla } is an activist organization led by the { -brand-name-mozilla-foundation } that makes change in the world through a variety of ventures including { -brand-name-mozilla-corporation }, MZLA, <a {$ventures}>{ -brand-name-mozilla-ventures }</a> and <a {$mozai}>{ -brand-name-mozilla-ai }</a>. How are we different? Because we’re mission-driven, it means we have the freedom to make all of our decisions based on what’s best for the internet and for everyone online, not based on the demands of shareholders—we don’t actually have any of those.
```

@reemhamz wdyt? I can add this here too…

Reply:
(or the opposite could be adding the spaces around the dash on L50 because https://github.com/mozilla-l10n/www-l10n/pull/404#discussion_r1614798808 …)

Reply:
Jumping in because Reem is out today. 

Seems like we should match L50 and use an em dash without spaces. I'll also check with our writers.

Reply:
And FWIW, that looks so wrong to me despite all the grammer blogs agreeing.

Reply:
Oh cool, I found [another place](https://github.com/mozilla/bedrock/pull/14559/files#diff-e0aa9deb46415c8f1eba2ab93093de1a3253765a2b3f9c6237ad101d18e71319) we do it in another way `shareholders — we don’t`.

Reply:
_(We're on the same boat I guess, I actually made the erroneous spacing intentionally in the comment about the correct usage 🤷…)_

I also don't like the way the line breaks around it as demonstrated by the sshot in the linked PR, but… yea. It looks so much more sophisticæted without the spaces, so snobbish!;)

Reply:
@stephaniehobson 👍 Anyways, you can always hit that "commit suggestion" button when you feel like including it with this… or not. I'm not sure how quickly this needs to go out, i.e. when a prod release is going to be needed so as not to release it with the Monitor typo. The cosmetic stuff can be sorted out separately, unless it means i10n impact.

That saying, maybe I should remove those: https://github.com/mozilla-l10n/www-l10n/pull/404#pullrequestreview-2079294790 so that they don't trigger unnecessary pontoon work?

Reply:
Okay, new plan, we keep the spaces on this one because that seems consistent with a few other places but we don't correct the other one until I hear back from the writers.